gitops/git/git.go

```diff
@@ -103,6 +104,34 @@ func (r *Repo) Commit(message, gitopsPath string) bool {
 	return true
 }
 
+// RemoveDiff removes the changes made to a specific file in the repository
+func (r *Repo) RemoveDiff(fileName string) {
```

Comment:
```suggestion
func (r *Repo) RestoreFile(fileName string) {
```
Align with the terminology used by Git. For example:
```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   src/index.ts
```